src/frequenz/channels/utils/timer.py

```diff
@@ -66,16 +66,18 @@ def reset(self) -> None:
     def stop(self) -> None:
         """Stop the timer.
 
-        Once `stop` has been called, all subsequent calls to `receive` will
-        immediately return None.
+        Once `stop` has been called, all subsequent calls to
+        [receive()][frequenz.channels.Timer.receive] will immediately return
+        None.
         """
         self._stopped = True
 
     async def receive(self) -> Optional[datetime]:
         """Return the current time once the next tick is due.
 
         Returns:
-            The time of the next tick.
+            The time of the next tick or `None` if it was
+                [stop()][frequenz.channels.Timer.stop]ed.
```

Comment:
The `stop()`ed looks a bit odd to me.  Do you like one of these better?

```
or `None` if `stop()` has been called on the timer
```

or 

```
or `None` if the timer has been [stopped][frequenz.channels.Timer.stop]
```